PubNub/Data/Service Objects/PNResult.m

```diff
@@ -157,7 +157,7 @@ - (NSDictionary *)normalizedServiceData:(id)serviceData {
     
     NSDictionary *normalizedServiceData = serviceData;
     if (serviceData && ![serviceData isKindOfClass:NSDictionary.class]) {
-        
+#warning information can be two things?
```

Comment:
@jzucker2 What do exactly you mean here? Normally only parsed data should reach here and it is always _NSDictionary_. In case if some message has been passed as error from NSURLSession or PubNub service itself which can't be parsed and represented as simple string - we will wrap it and such object will become _error_.